lib/awful/client.lua

```diff
@@ -999,11 +999,22 @@ function client.is_transient_for(c, c2)
 end
 
 -- Register standards signals
+
+--- The last geometry when client was floating.
+-- @signal property::floating_geometry
 capi.client.add_signal("property::floating_geometry")
+
 capi.client.add_signal("property::floating")
+
 capi.client.add_signal("property::dockable")
+
+--- The client marked signal.
+-- @signal .marked
 capi.client.add_signal("marked")
-capi.client.add_signal("unmarked")
+
+--- The client unmarked signal.
+-- @signal unmarked
+capi.client.add_signal(".unmarked")
```

Comment:
Ah, I missed one. At first I remove all the leading points form the doc, then figured out that ldoc suck and it was why they were there. Then added them back using sed, but forgot to re-re-fix this.

> (Also, why the heck do we have marked and unmarked and not just property::marked...)

the first 2 are deprecated and documented as such. At some point we need to add add_signal_deprecated to that all `connect_signal()` print a warning.